nms/packages/magmalte/app/components/NetworkSelector.js

```diff
@@ -26,9 +26,9 @@ import React from 'react';
 import SettingsEthernetIcon from '@material-ui/icons/SettingsEthernet';
 import Text from '../../fbc_js_core/ui/components/design-system/Text';
 import Tooltip from '@material-ui/core/Tooltip';
-import classNames from 'classnames';
 import useMagmaAPI from '../../api/useMagmaAPI';
 
+import symphony from '../../fbc_js_core/ui/theme/symphony';
```

Comment:
You'll need to restructure `fbc_js_core/ui/theme` a bit.

For context, 'symphony' was an old project that was discontinued, and now partially lives on as a zombie within Magma (or fbc-js-core). We don't really want any references to a 'symphony' to continue living on.

Reply:
I'll use `app/theme/default` instead and I'll work on a separate PR to get rid of `fbc_js_core/ui/theme`.